core/src/main/java/com/linecorp/armeria/internal/common/NettyFutureUtil.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.common;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.concurrent.CompletableFuture;
+
+import io.netty.util.concurrent.Future;
+
+public final class NettyFutureUtil {
+
+    public static <A> CompletableFuture<A> toCompletableFuture(Future<A> future) {
```

Comment:
How about merging this into `UnmodifiableFuture.fromNetty(Future<T>)` ?

Reply:
Oh... nice suggestion!